MotuClUploadCMEMSPhysModel.py

```diff
@@ -0,0 +1,65 @@
+import argparse
+import pandas as pd
+from pathlib import Path
+import sys, os
+
+def change_time_format(timemin,timemax):
```

Comment:
Personally, I would have gone for [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) notation of time: `{yyyy}-{MM}-{dd}T{HH}:{mm}:{SS}Z`